src/fused_api.cpp

```diff
@@ -35,6 +35,24 @@
 #include <miopen/logger.hpp>
 #include <miopen/tensor.hpp>
 
+enum class ConvDirection
+{
+    Fwd = 1,
+    Bwd = 2,
+    WrW = 4
+};
+
+namespace miopen {
+namespace debug {
+void LogCmdConvolution(const miopenTensorDescriptor_t& xDesc,
```

Comment:
This is a forward declaration to use the function in convolution_api.cpp?

Reply:
yes! Should I write this in comment to make it more clear?